runtime/oti/j9nonbuilder.h

```diff
@@ -44,6 +44,8 @@
 #define CLASSNAME_VALID_NON_ARRARY	1
 #define CLASSNAME_VALID_ARRARY		2
 
+#define  J9RecompDueToInlinedMethodRedefinition 0x000C0000
```

Comment:
Does this belong here? If so, maybe all of the reason constants should move here.